packages/core/src/texture/RenderTarget.ts

```diff
@@ -153,320 +149,74 @@ export class RenderTarget extends EngineObject {
-
-    this._frameBuffer = null;
-    this._colorTextures.length = 0;
-    this._depthTexture = null;
-    this._depthRenderBuffer = null;
-    this._MSAAFrameBuffer = null;
-    this._MSAAColorRenderBuffers.length = 0;
-    this._MSAADepthRenderBuffer = null;
-  }
-
-  /**
-   * Activate this RenderTarget.
-   * @remarks
-   * If MSAA is turned on, MSAA FBO is activated, and then this._blitRenderTarget() is performed to exchange FBO.
-   * If MSAA is not turned on, activate the main FBO.
-   */
-  public _activeRenderTarget(): void {
-    const gl: WebGLRenderingContext & WebGL2RenderingContext = this._rhi.gl;
-
-    if (this._MSAAFrameBuffer) {
-      gl.bindFramebuffer(gl.FRAMEBUFFER, this._MSAAFrameBuffer);
-    } else {
-      gl.bindFramebuffer(gl.FRAMEBUFFER, this._frameBuffer);
-    }
-  }
-
   /**
    * Set which face of the cube texture to render to.
    * @param faceIndex - Cube texture face
    */
-  public _setRenderTargetFace(faceIndex: TextureCubeFace = TextureCubeFace.PositiveX): void {
-    const gl: WebGLRenderingContext & WebGL2RenderingContext = this._rhi.gl;
-    const colorTexture = this._colorTextures[0];
-    const depthTexture = this._depthTexture;
-
-    gl.bindFramebuffer(gl.FRAMEBUFFER, this._frameBuffer);
-
-    // Bind render color texture
-    if (colorTexture?._isCube) {
-      gl.framebufferTexture2D(
-        gl.FRAMEBUFFER,
-        gl.COLOR_ATTACHMENT0,
-        gl.TEXTURE_CUBE_MAP_POSITIVE_X + faceIndex,
-        colorTexture._glTexture,
-        0
-      );
-    }
-
-    // Bind depth texture
-    if (depthTexture?._isCube) {
-      gl.framebufferTexture2D(
-        gl.FRAMEBUFFER,
-        depthTexture._formatDetail.attachment,
-        gl.TEXTURE_CUBE_MAP_POSITIVE_X + faceIndex,
-        depthTexture._glTexture,
-        0
-      );
-    }
-
-    // Revert current activated render target
-    this._activeRenderTarget();
+  setRenderTargetFace(faceIndex: TextureCubeFace): void {
+    this._platformRenderTarget.setRenderTargetFace(faceIndex);
   }
 
   /**
    * Blit FBO.
    */
-  public _blitRenderTarget(): void {
-    const gl: WebGLRenderingContext & WebGL2RenderingContext = this._rhi.gl;
-    const mask = gl.COLOR_BUFFER_BIT | (this._depthTexture ? gl.DEPTH_BUFFER_BIT : 0);
-    const colorTextureLength = this._colorTextures.length;
-
-    gl.bindFramebuffer(gl.READ_FRAMEBUFFER, this._MSAAFrameBuffer);
-    gl.bindFramebuffer(gl.DRAW_FRAMEBUFFER, this._frameBuffer);
-
-    for (let textureIndex = 0; textureIndex < colorTextureLength; textureIndex++) {
-      const attachment = gl.COLOR_ATTACHMENT0 + textureIndex;
-
-      this._blitDrawBuffers[textureIndex] = attachment;
-
-      gl.readBuffer(attachment);
-      gl.drawBuffers(this._blitDrawBuffers);
-      gl.blitFramebuffer(0, 0, this._width, this._height, 0, 0, this._width, this._height, mask, gl.NEAREST);
-
-      this._blitDrawBuffers[textureIndex] = gl.NONE;
-    }
-
-    gl.bindFramebuffer(gl.FRAMEBUFFER, null);
+  blitRenderTarget(): void {
```

Comment:
For internal function you must add internal comments and add "_" for function name.